_posts/2017-10-30-lambda-world-conference.md

```diff
@@ -0,0 +1,73 @@
+
+<center>
+<img src="{{site.baseurl}}/assets/img/custom/blog/2017-10-30-lambda-world-conference/hylomorphism.png" alt="Hylomorphism" class="img img-responsive" width="80%"/>
+<small>Slide from <a href="http://slides.com/zainabali_/peeling_the_banana#/" target="_blank">'Peeling the Banana - Recursion schemes from first principles'</a> by Zainab Ali</small>
+</center>
+<br/>
+
+**Jeremy Gibbons**, professor of Computing at the University of Oxford, was the star of the final keynote, showing the relation between the structure of multi-dimensional data and Naperian functors.
+
+Do you know another technical conference which provides its own beer, _functional_ beer?
+
+<center>
+<img src="{{site.baseurl}}/assets/img/custom/blog/2017-10-30-lambda-world-conference/functional-beer.jpg" alt="Functional Beer" class="img img-responsive"/>
+</center>
+<br/>
+
+Organizers deserve my warmest congratulations, because it was one of the best conferences I've attended. It was the third edition of this awesome event and, despite the high number of attendees, it kept its high quality, the personal attention and its family atmosphere. <a href="https://twitter.com/Jorge__Galindo">**Jorge Galindo**</a> is one of the organizers and his daughter is growing at the same time as the event, getting used to face a huge audience:
+
+<center>
+<img src="{{site.baseurl}}/assets/img/custom/blog/2017-10-30-lambda-world-conference/jorge-galindo-s-daughter.jpg" alt="Jorge Galindo" class="img img-responsive" width="50%"/>
+</center>
+<br/>
+
+I'll update this post when videos are available.
+
+## Acknowledgments
+
+My special thanks go to my colleagues **Christian Panadero** and <a href="/publications/author/carlos-morera%20de%20la%20chica/">**Carlos Morera de la Chica**</a>, here with <a href="https://bartoszmilewski.com/" target="_blank">**Bartosz Milewski**</a>, the author of <a href="https://bartoszmilewski.com/2014/10/28/category-theory-for-programmers-the-preface/" target="_blank">_Category Theory for Programmers_</a>:
+
+<center>
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Such a moment meeting <a href="https://twitter.com/BartoszMilewski?ref_src=twsrc%5Etfw">@BartoszMilewski</a> in <a href="https://twitter.com/Lambda_World?ref_src=twsrc%5Etfw">@Lambda_World</a>. Looking forward to our next category theory session at <a href="https://twitter.com/codurance?ref_src=twsrc%5Etfw">@codurance</a> next week. <a href="https://t.co/QefB9DXAhG">pic.twitter.com/QefB9DXAhG</a></p>&mdash; Carlos Morera (@CarlosMChica) <a href="https://twitter.com/CarlosMChica/status/924317459859103749?ref_src=twsrc%5Etfw">October 28, 2017</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+</center>
+
+They're awesome _travel_ companions in this journey and I met a lot of interesting people thanks to them.
```

Comment:
They were instead of they are?

Reply:
You were, you are and you will be ;-) I'm referring to the journey of learning/improving...